### PR TITLE
Fix/tooltip delegates

### PR DESCRIPTION
**Pull request by rafinskipg**
Adds a little tooltip on participation metrics for delegates

### CODE REVIEW

File: pages/executive/[proposal-id].tsx [hunk 1]
```diff
@@ -254,7 +254,7 @@ const ProposalView = ({ proposal }: Props): JSX.Element => {
                 tabListStyles={{ pl: [3, 4] }}
                 tabTitles={['Spell Details']}
                 tabPanels={[
-                  <div key={1} sx={{ p: [3, 4] }}>
+                  <div key={'spell'} sx={{ p: [3, 4] }}>
```
Comment (adamgoth) [inline pages/executive/[proposal-id].tsx:257]: Should these `key` changes be included?

File: modules/delegates/components/DelegateCard.tsx [hunk 2]
```diff
@@ -33,22 +37,6 @@ export function DelegateCard({ delegate }: PropTypes): React.ReactElement {
...
-  );
-  const communicationTooltipLabel = (
-    <>
-      The percentage of votes for which the delegate has publicly <br />
-      communicated their reasoning in addition to voting. <br />
-      Combines stats for polls and executives. <br />
-      Updated weekly by the GovAlpha Core Unit. <br />
-    </>
-  );
-
```
Comment (adamgoth) [inline modules/delegates/components/DelegateCard.tsx:51]: I moved these to the participation metrics component so we can reuse label text

**Review by adamgoth — APPROVED** [review]
LGTM but wondering if these `key` changes should be included?

File: pages/executive/[proposal-id].tsx (hunk 1 above)
Comment (rafinskipg) [reply pages/executive/[proposal-id].tsx:257]: they don't hurt, i almost prefer them to the indexed key that can fail when moving components

Comment (adamgoth) [reply pages/executive/[proposal-id].tsx:257]: Just wanted to make sure if it was intended for this PR but I agree 👍

**Pull request merged**